pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -225,7 +225,10 @@ func encodeVCToJSON(vcBytes []byte, testFileName string) {
 func encodeVPToJSON(vcBytes []byte) {
 	// https://www.w3.org/TR/vc-data-model/#presentations-0 states "If present" under verifiableCredential
 	// but the test suite requires the element to be present. Hence, WithPresRequireVC is used in test suite runs.
-	vp, err := verifiable.ParsePresentation(vcBytes, verifiable.WithPresRequireVC())
+	vp, err := verifiable.ParsePresentation(vcBytes,
+		verifiable.WithPresRequireVC(),
+		verifiable.WithPresRequireProof(),
+		verifiable.WithDisabledPresentationProofCheck())
```

Comment:
We are requiring proof yet disabling the proof check...

Reply:
We use `WithDisabledPresentationProofCheck()` to disable a check of the signature of the proof. This is applied in the VC test suite as their verification methods are invalid (e.g. DIDs are unresolvable). We apply the same approach for the VC.
As for `WithPresRequireVC()` and  `WithPresRequireProof()` options, they are used to insist on presence of `verifiableCredential` and `proof` fields in the VP. This is a sort of workaround to pass the test cases. In the https://www.w3.org/TR/vc-data-model spec, for example, the presence of `verifiableCredential` is not mandatory.

Reply:
@kdimak - I agree with @llorllale , it's confusing. Few questions would be -  which would take precedence (does it check vp proof or not?) when both are set (`WithPresRequireProof` and `WithDisabledPresentationProofCheck`) ? Does the order matter ? what happens when none of those are set ?

I feel we should have a default behaviour and then one option to override it.

Reply:
@kdimak it now looks like this:

```go
	vp, err := verifiable.ParsePresentation(vcBytes,
		verifiable.WithPresRequireVC(),
		verifiable.WithPresRequireProof(),
		verifiable.WithPresDisabledProofCheck())
```

Trying to understand what you're saying... `WithPresRequireProof()` merely checks the presence of the `proof` attribute, while `WithPresDisabledProofCheck()` disables verification of the `proof` element (if present). Is that it?

Reply:
In that case, the VC just has one option. https://github.com/hyperledger/aries-framework-go/blob/694a5b6c0c217f18ff84f47ddbd9ca598d26455f/pkg/doc/verifiable/credential.go#L589-L593 
Any specific reason to separate proof attribute check and validate the proof in VP only?

Reply:
1. `WithDisabledProofCheck()` for VCs and `WithPresDisabledProofCheck()` for VPs serve the same purpose: we want to avoid checking crypto primitives (as this check will lead to the error due to unresolvable DIDs in VC test suite data).
2. `WithPresRequireProof()` is used to pass one of the test cases from the VC test suite to have a mandatory `proof` element that [seems incorrect](https://www.w3.org/TR/vc-data-model/#presentations-0). For example, we can parse VP in JWS form, and thus it does not require to have a `proof` element (it's called an embedded proof) since we already verified VP's external proof (a signature from JWT).